examples/problems/1100_callback.ipynb

```diff
@@ -0,0 +1,564 @@
+{
```

Comment:
<p>we don't get a joint cost. we get a joint embedding.</p><p> </p><p>Say "To create a joint PCA embedding between two sets of genes, we compute the PCA embedding for pairs of distributions by passing [and now what you said]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>